pkg/kubelet/kubelet_pods.go

```diff
@@ -468,8 +468,6 @@ func (kl *Kubelet) GenerateRunContainerOptions(pod *v1.Pod, container *v1.Contai
 	podName := volumeutil.GetUniquePodName(pod)
 	volumes := kl.volumeManager.GetMountedVolumesForPod(podName)
 
-	opts.PortMappings = kubecontainer.MakePortMappings(container)
```

Comment:
simplifying code review: This is not needed as `GenerateRunContainerOptions` generates options for container while port mapping is for an entire pod. The same method is called from `generatePodSandboxConfig` where it will be used.